src/language/en/en.json

```diff
@@ -92,7 +92,8 @@
     "fee": "Fee",
     "successTransaction": "Success transaction",
     "placeholder": "This Percentage",
-    "help": "Help"
+    "help": "Help",
+    "maxValueAllowed": "The max value allowed is 100%"
```

Comment:
I beleive the max value for a fee is 3%

![image](https://user-images.githubusercontent.com/5632966/95001600-a7923480-0588-11eb-91a8-c2ff3e8ae121.png)